### PR TITLE
Add options to eslintrc.js.

### PR DESCRIPTION
**Pull request by knightburton**
IoT.jsCode-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

**Comment by knightburton** [comment]
I've added a few options to the eslintrc file which we should use in my opinion. These options are not carved in stone, e.g.: I've setted the code line length to 120 character because I think the 80 character is a bit old "approach", but this is just a brave idea.

File: .eslintrc.js [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+    "browser": true,
+    "node": true,
+    "es6": true,
+    "jquery": true
+  },
+  "extends": "eslint:recommended",
+  "rules": {
+    "no-console": "off",
+    "no-var": 2,
+    "space-before-function-paren": ["error", "always"],
```
Comment (daeyeon) [inline .eslintrc.js]: Looking at IoT.js, members look to prefer writing no space before function parenthesis. I'm also a bit inclined to that.

**Comment by knightburton** [comment]
@daeyeon I've updated the PR, but should we wait for the end of thecoding convention discussion or not?

**Comment by yichoi** [comment]
LGTM

**Pull request merged**